src/components/aboutUs/aboutUs.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+//import PropTypes from 'prop-types';
+import "./aboutUs.css";
+
+const aboutUs = () => {
```

Comment:
Typically because this const will need to be identified with a capital letter (`AboutUs`) when it's imported (for us in `index.js`), it's also capitalized here on declaration.  Do you have a reason for _not_ capitalizing it here? It may be that how I've typically seen it done is not the most correct way.

Reply:
@kdv24 I think what you said is correct. I had no idea it was best practice to capitalize components on declaration. I was told from my mentor to get use to camel casing when declaring variables but it looks like components is an exception. Thanks for the note!

Reply:
👍